low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -183,6 +183,13 @@ def config_sspl(self):
             if not os.path.exists(consts.REPLACEMENT_NODE_ENV_VAR_FILE):
                 ServiceV().validate('isrunning', ['consul'], is_process=True)
 
+        # Overwrite sspl_test config by the copy of global config
```

Comment:
This approach is fine as far as backend global config is maintained & shared in a file, so ok to unblock now. 
Keep a TODO though to make use of ConfStor APIs to make copy of whole global config & use, irrespective of its backend type.

Reply:
or i think we can even get a dump of config in format we need using ConfStor APIs, try that .

Reply:
yes, conf.copy helps.